app/components/primer/beta/auto_complete/auto_complete.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= render Primer::BaseComponent.new(**@system_arguments) do %>
-  <label for="<%= @input_id %>" class="<%= @label_classes %>">
+  <label for="<%= @input_id %>" class="<%= @label_classes if @is_label_visible %>">
```

Comment:
This should work just fine, but what about setting `@label_classes` to `nil` in the Ruby code?

Reply:
So, I looked at that 😄 .

Currently label class is set in a ternary, so adding another conditional would be a bigger code change. If you think it's clearer that way I'm happy to update it 👍